docs/writing_articles.md

```diff
@@ -0,0 +1,119 @@
+# Writing articles
```

Comment:
Couple of things here: it is mildly unclear if "articles" here refers simply to the news articles, or the static pages as well. This is really clear in the markdown section, where it's clearly news, but in the html section it just says "more complex articles" but doesn't really say whether those are still news articles. 

I tried writing an html *news* article following the the docs as well as i can manage and am getting the following error when I run it:
```
jinja2.exceptions.TemplateNotFound

jinja2.exceptions.TemplateNotFound: content/news/2021-09-19-html-test.html.jinja.html.jinja2
```
and I cannot tell if this is a bug, or the documentation being unclear